http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreterSpec.scala

```diff
@@ -124,46 +111,6 @@ class DataprocInterpreterSpec
     clusterCreationRes.serviceAccountKey shouldBe None
   }
 
-  it should "resize cluster when stopping cluster" in isolatedDbTest {
```

Comment:
do we have resize test somewhere still?

Reply:
added a stop/start auto test in https://github.com/DataBiosphere/leonardo/pull/1812/commits/fd17d266f75c0daa5bc9a63d8ac9a8d9b9dab8a9